config/routes.rb

```diff
@@ -1,16 +1,12 @@
 Rails.application.routes.draw do
   devise_for :users
-  root 'welcome#index'
+  root 'posts#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.